pages/glossary/programmatic-section-of-content.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Programmatic section of content
+key: programmatic-section-of-content
+unambiguous: true
+objective: true
+---
+
+A _programmatic section of content_ is either an [explicit section of content][] or an [implicit section of content][].
+
+Programmatic sections of content may contain nested programmatic subsections of content.
+
+Programmatic sections of content may have zero or one heading [associated][] with them.
+
+**Note:** This definition roughly follows the [algorithm for creating an outline][outline algorithm], but using different elements for defining sections. The main focus is here on the [semantic role][] rather than the element itself, and all [landmark roles][landmark] are considered. Additionally, programmatic sections of content include content from [shadow trees][] and [nested browsing contexts][] which is ignored by the [algorithm for creating an outline][outline algorithm].
+
+**Note:** When elements and roles are used according to their semantics, programmatic sections of content and [sections of content][] should match perfectly.
+
+#### Assumption
+
+- This definition assumes that the [semantic roles][semantic role] (including the level of headings) are used according to their semantics.
+- This definition assumes that the `hgroup` element is not used.
+
+#### Examples
+
+**Note:** The examples presented here are non-normative and not testable. They serve to illustrate some common pitfalls about the definition and to help implementers of ACT rules understand it.
```

Comment:
Saying the examples are non-normative gives the impression the rest of this is normative, which it isn't. I think this should be rephrased.